SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/UNR-3761/SpatialTestHandover/SpatialTestHandover.cpp

```diff
@@ -36,96 +36,26 @@ ASpatialTestHandover::ASpatialTestHandover()
 	Author = "Andrei";
 	Description = TEXT("Test Actor handover");
 
-	SpawnLocation = FVector(-500.0f, -500.0f, 50.0f);
-
-	TestLocations.Add(FVector(500.0f, -500.0f, 50.0f));
-	TestLocations.Add(FVector(500.0f, 500.0f, 50.0f));
-	TestLocations.Add(FVector(-500.0f, 500.0f, 50.0f));
+	Server1Position = FVector(-500.0f, -500.0f, 50.0f);
```

Comment:
Hmm, Mike, you will know, but is this guaranteed that the virtual worker ids will always get assigned to these zones, and should we be relying on it in a test. My intuition is that we should e.g. be using marker entities for each zone so we can say "have the same authority as this entity", but it does make the test a lot more complicated.

Reply:
By definition the virtual ids are always the same for the same LBS, however if you replace it with another one (or if in the future people decide to change the virtual id algorithm) it will not work.

One can change the test to have anchors in the world instead that you need to drag in the inspector, but the thing here that is trickier is that we're doing steps on specific workers.

I think that the most pressing thing is that we need a GetVirtualWorkerIdFor( FVector ) function.

Reply:
> Hmm, Mike, you will know, but is this guaranteed that the virtual worker ids will always get assigned to these zones, and should we be relying on it in a test.

It's up to the LBS - but yes currently the grid based LBS will do this.

> I think that the most pressing thing is that we need a GetVirtualWorkerIdFor( FVector ) function.

I can see the value here, unfortunately LBS will often require more information than just a position to determine which virtual worker id should have authority, so we don't want to expose that as a general purpose function for the strategy.

Reply:
Based on this conversation, did this in case we'd want it https://github.com/spatialos/UnrealGDK/pull/2589